pkg/tsdb/prometheus/go.mod

```diff
@@ -0,0 +1,136 @@
+module github.com/grafana/grafana/pkg/tsdb/prometheus
```

Comment:
I don't think the whole DS will become a module? I imagine there is some stub in tsdb/prometheus that references most of the code (which maybe lives on level deeper?) that is the "core/vanilla prom ds"... that make sense?

Reply:
This is just a test to see how CI is reacting. We will close this when we are done with testing.